apps/backoffice-v2/src/lib/blocks/components/AmlBlock/hooks/useAmlBlock/useAmlBlock.tsx

```diff
@@ -78,264 +86,104 @@ export const useAmlBlock = (data: Array<TWorkflowById['context']['aml']>) => {
                   },
-                })
-                .addCell({
-                  type: 'table',
-                  value: {
-                    props: {
-                      table: {
-                        className: 'my-8',
-                      },
-                    },
-                    columns: [
-                      {
-                        accessorKey: 'entry',
-                        header: 'Adverse Media',
-                        cell: props => {
-                          const value = props.getValue();
+                  header: 'Sanctions',
+                }),
+                columnHelper.accessor('adverseMedia', {
+                  cell: info => {
+                    const adverseMediaLength = info.getValue().length;
 
-                          return (
-                            <div className={'flex space-x-2'}>
-                              <WarningFilledSvg className={'mt-px'} width={'20'} height={'20'} />
-                              <span>{value}</span>
-                            </div>
-                          );
-                        },
-                      },
-                      {
-                        accessorKey: 'date',
-                        header: 'Date',
-                      },
-                      {
-                        accessorKey: 'source',
-                        header: 'Source URL',
-                        cell: props => {
-                          const value = props.getValue();
+                    return <TextWithNAFallback>{adverseMediaLength}</TextWithNAFallback>;
+                  },
+                  header: 'Adverse Media',
+                }),
+                columnHelper.accessor('fitnessProbity', {
+                  cell: info => {
+                    const fitnessProbityLength = info.getValue().length;
 
-                          return (
-                            <a
-                              className={buttonVariants({
-                                variant: 'link',
-                                className: 'h-[unset] cursor-pointer !p-0 !text-blue-500',
-                              })}
-                              target={'_blank'}
-                              rel={'noopener noreferrer'}
-                              href={value}
-                            >
-                              Link
-                            </a>
-                          );
-                        },
-                      },
-                    ],
-                    data: adverseMedia,
+                    return <TextWithNAFallback>{fitnessProbityLength}</TextWithNAFallback>;
                   },
-                })
-                .build()
-                .flat(1),
-            })
-            .build()
-            .flat(1),
-        ) ?? []),
+                  header: 'Fitness Probity',
+                }),
+              ],
+            },
+          })
+          .build()
+          .flat(1),
```

Comment:
Is the `?? []` no longer needed?

Reply:
The `?? []` was part of running over the `matches` array, which we no longer do the same way, hence not needed